packages/mongo-nav/package.json

```diff
@@ -22,6 +22,7 @@
     "@leafygreen-ui/badge": "^3.0.0",
     "@leafygreen-ui/button": "^4.0.0",
     "@leafygreen-ui/hooks": "^2.0.0",
-    "@leafygreen-ui/icon-button": "^4.0.0"
+    "@leafygreen-ui/icon-button": "^4.0.0", 
+    "@leafygreen-ui/leafygreen-provider": "1.0.0"
```

Comment:
Let's be consistent about our versioning:
```
"@leafygreen-ui/leafygreen-provider": "^1.0.0"
```

Reply:
👍